Space-Runners/ablo-demo-fe
# Cu 863hc3d9k/update prompts and keyword suggestions replace scrap collage style with william morris style/bojan tosic

### Description
- Replace 'Scrap collage' style with 'William Morris' style

### Testing Instructions
- Backend has to be merged first to see this change
- New William Morris style will replace Scrap Collage
- New suggestions for most styles
